api/services/control/control.proto

```diff
@@ -60,8 +60,8 @@ message UsageRecord {
 message SolveRequest {
 	string Ref = 1;
 	pb.Definition Definition = 2;
-	string Exporter = 3;
-	map<string, string> ExporterAttrs = 4;
+	string ExporterDeprecated = 3;
```

Comment:
Could you document what happens when both `Exporter` and `Exporters` are set?
Can be a follow-up.

Reply:
Done.